common/djangoapps/adaptive_learning/config/tests/test_models.py

```diff
@@ -27,10 +27,10 @@ def setUp(self):
               (False, True, False),
               (True, True, True))
     @ddt.unpack
-    def test_method_feature_enabled(self, global_enabled_flag, course_enabled_flag, should_course_be_enabled):
+    def test_method_feature_enabled(self, global_enabled_flag, course_enabled_flag, feature_enabled_for_course):
         with adaptive_learning_enabled_feature_flags(global_enabled_flag,
                                                      self.course_id,
                                                      course_enabled_flag):
             self.assertEqual(
-                    AdaptiveLearningEnabledFlag.feature_enabled(self.course_id),
-                    cshould_course_be_enabled)
+                AdaptiveLearningEnabledFlag.feature_enabled(self.course_id),
+                feature_enabled_for_course)
```

Comment:
@tomaszgy Please move the closing parenthesis to the next line, to match the style guide for [breaking long lines](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines):

> Closing paren should be on a line by itself, indented the same as the first line.

Reply:
👍